tests/unit/tools/executor/test_tool_executor_more.py

```
@@ -144,7 +144,12 @@ def test_main_loop_get_tool_raises_appends_failure(
 
     ok = ToolResult(name="black", success=True, output="", issues_count=0)
 
-    def fake_get_tools(tools: str | None, action: str) -> ToolsToRunResult:
+    def fake_get_tools(
+        tools: str | None,
```

Ruff (0.15.6)

[warning] Unused function argument: `tools`

(ARG001)